src/append/rolling_file/policy/compound/trigger/mod.rs

```diff
@@ -1,18 +1,22 @@
 //! Triggers
 
-use std::{error::Error, fmt};
+use std::error::Error;
+use std::fmt;
+use std::option::Option;
 
 use crate::append::rolling_file::LogFile;
 #[cfg(feature = "file")]
 use crate::file::Deserializable;
 
 #[cfg(feature = "size_trigger")]
 pub mod size;
+#[cfg(feature = "time_trigger")]
+pub mod time;
 
 /// A trait which identifies if the active log file should be rolled over.
 pub trait Trigger: fmt::Debug + Send + Sync + 'static {
     /// Determines if the active log file should be rolled over.
-    fn trigger(&self, file: &LogFile) -> Result<bool, Box<dyn Error + Sync + Send>>;
+    fn trigger(&self, file: Option<&LogFile>) -> Result<bool, Box<dyn Error + Sync + Send>>;
```

Comment:
We cant make a breaking change like this

Reply:
I'll revert the change.